syntax = "proto3";

package patient;

service PatientService {
  rpc GetPatientInfo (PatientRequest) returns (PatientResponse);
}

message PatientRequest {
  string patient_id = 1;
}

message PatientResponse {
  string name = 1;
  int32 age = 2;
  string status = 3;
}
